gammapy/spectrum/models.py

```diff
@@ -2,24 +2,66 @@
 """Spectral models for Gammapy.
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
+from ..extern.bunch import Bunch
+from . import CountsSpectrum
+import numpy as np
+
 
 __all__ = [
     'SpectralModel',
     'PowerLaw',
     'ExponentialCutoffPowerLaw',
 ]
 
+# Note: Consider to move stuff from _models_old.py here
 
 class SpectralModel(object):
     """Spectral model base class.
     """
+    def __call__(self, energy):
+        kwargs = self.parameters
+        kwargs.update(energy=energy)
+        return self.evaluate(**kwargs)
 
 
 class PowerLaw(SpectralModel):
-    """Spectral power-law model.
+    r"""Spectral power-law model.
+    
+    .. math:: 
+
+        F(E) = F_0 \cdot \left( \frac{E}{E_0} \right)^{-\Gamma}
+
+    Parameters
+    ----------
+    index : float, `~astropy.units.Quantity`
+        :math:`\Gamma`
+    amplitude: float, `~astropy.units.Quantity` 
+        :math:`F_0`
+    reference : float, `~astropy.units.Quantity` 
+        :math:`E_0`
     """
+    def __init__(self, index, amplitude, reference):
+        self.parameters = Bunch(index = index,
```

Comment:
I think I like the idea of storing `parameters` in a `Bunch`.
Maybe rename to `pars`? (either way is fine with me)

Can you mention this somewhere, maybe the docstring of the base class?